frontend/src/layouts/BaseLayout.vue

```diff
@@ -223,6 +223,21 @@
               option-label="label"
               rounded
             />
+
+            <!-- Clear local storage button -->
+            <div class="q-mt-md row items-center">
```

Comment:
Let's hide the clear storage button unless the advanced mode is on.

Reply:
might be confusing because the advanced mode doesnt utilize the caching functionality? only the default fetching/scanning functionality has been updated to incorporate caching.

Reply:
Hmm, didn't realize it wasn't supported in advanced mode. I guess it makes sense that we don't. 

Could we move it above the language selector then?

Reply:
https://github.com/ScopeLift/umbra-protocol/pull/685/commits/ba09324b2bb07d4115de28ac9bcbe3dcf9f07ec1

Reply:
Sorry for the back and forth, but if clearing the local storage is also inevitably resetting the send history, could we just make the current "Send history on/off" toggle clear the local storage when it's turned off? We can rename the toggle to "Send and Announcement history" or something and update the `?` description accordingly.

The extra clear storage button just seems so out of place no matter where it is placed.

Reply:
Adding a toggle like "Send and Announcement History On" would add a bit more complexity to the current announcement caching and scanning logic. I'd rather have the send history and announcement handling be separate concerns for now.

Reply:
Shouldn't it be just a change of:

`{{ sendHistorySave ? $t('Base-Layout.on') : $t('Base-Layout.off') `**&& clearLocalStorage()**` }}</span`

I haven't fully convinced myself that combining those two features is a good idea either, so yea, maybe we should leave it for now.

Reply:
Mmm not sure that would work. 

For clearing send history it's simply clearing local storage when the toggle is off. But if we need similar functionality for announcements, ie. announcement toggle is off, then will need to add corresponding logic to the AccountReceive component and update the default scan to not save user announcements, etc.

Not sure if worth pursuing.